deploy-as-code/helm/environments/ci-secrets.yaml

```
@@ -2177,6 +2177,39 @@ cluster-configs:
                                       value: ENC[AES256_GCM,data:1IgHKOx7l/9oHUvnhoD5VNLST4TC95h/OSGe3HPUb1UmpIDA4jrn3Q==,iv:iYr3wGDghvnm4RfNsQEnSn7+qc7mEZQ6NNc+76kDqG4=,tag:8y4jUsxUtKzG+Yq8BSy2Ww==,type:str]
                                     - name: ENC[AES256_GCM,data:5IMHt772W5d/BA==,iv:LA3qKgWkenFYfb+ZKpdDuUfBR2iUWxQNQs7HV5T1mho=,tag:Rjj3fyomjgRvfghxerN4Wg==,type:str]
                                       value: ENC[AES256_GCM,data:sj9nyxnS6Re+VQ==,iv:bdoQbPfkmHqCaBn6KpjWKUJyIf8hglxbajTNMieEou0=,tag:5o5Nzficj2HuBIEwk4lZvA==,type:str]
+                unified-moz-qa:
+                    apiVersion: ENC[AES256_GCM,data:/yE=,iv:ktHgMs8dYqO8tG2GDSCVt2DxcdGZU4NKZk75i/zRKIE=,tag:wGLu6NJv/3bPivFTShSxHw==,type:str]
+                    preferences: {}
+                    kind: ENC[AES256_GCM,data:l8T4cV3x,iv:WXotFERS4FoCLMPcqGYJ8N6WseaOJgqXrMvFAC7PatQ=,tag:q6ksvyUEfHirumBDhnQBDQ==,type:str]
+                    clusters:
+                        - cluster:
+                            server: ENC[AES256_GCM,data:K6HqoO8WU2IxQ1YUB4dbTZEQ0vdRouaFT5eRNfIYFN9ypgX7DoWZLmRHFiLeR0yxQfOeHdfx3E54KfeKnh0AN3ojv3lHSeAk2Q==,iv:+M4Pb/qWDaYR4UoRwEIvm2JQtLVgGIsLFB1y8UHMvbg=,tag:7YHCsYBr7TdvMwhF8LOdkA==,type:str]
```

yamllint (1.35.1)

[warning] wrong indentation: expected 30 but found 28

(indentation)